cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -232,7 +232,8 @@ def format_hash(x): return hashlib.md5(
         return format_hash(json.dumps(o))
 
     def calculate_job_id(self, par_dic):
-        return u'%s' % (self.make_hash(par_dic))
```

Comment:
I am not too sure why we have both `make_hash` and `calculate_job_id` but ok, it's not part of this PR.

Reply:
`calculate_job_id` is used when generating a `job_id` starting from the dictionary of the request, and during the vavlidation of the `job_id` (calculation from the `analysis_parameters.json` file)

Reply:
Another copy of the make_hash is in the pytests_fixtures

Reply:
yes, very good.

Reply:
yes, very good.